content/en/docs/reference/using-api/api-overview.md

```diff
@@ -18,95 +18,115 @@ This page provides an overview of the Kubernetes API.
+    re-creating API objects. The editing process may not be straightforward.
+    The migration may require downtime for applications that rely on the feature.
+  - The software is not recommended for production uses. Subsequent releases
+    may introduce incompatible changes. If you have multiple clusters which
+    can be upgraded independently, you may be able to relax this restriction.
+  
+  {{< note >}}
+  Try the beta features and provide feedback. After the features exit beta, it
+  may not be practical to make more changes.
+  {{< /note >}}
+  
 - Stable:
   - The version name is `vX` where `X` is an integer.
   - The stable versions of features appear in released software for many subsequent versions.
 
 ## API groups
 
-[API groups](https://git.k8s.io/community/contributors/design-proposals/api-machinery/api-group.md) make it easier to extend the Kubernetes API. The API group is specified in a REST path and in the `apiVersion` field of a serialized object.
+[API groups](https://git.k8s.io/community/contributors/design-proposals/api-machinery/api-group.md)
+make it easier to extend the Kubernetes API.
+The API group is specified in a REST path and in the `apiVersion` field of a
+serialized object.
 
 Currently, there are several API groups in use:
 
-- The *core* (also called *legacy*) group is found at REST path `/api/v1`. The core group is not specified
-  as part of the `apiVersion` field, for example, `apiVersion: v1`.
-- The named groups are at REST path `/apis/$GROUP_NAME/$VERSION` and use `apiVersion: $GROUP_NAME/$VERSION`
-  (for example, `apiVersion: batch/v1`). You can find the full list of supported API groups in
-  [Kubernetes API reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/).
+*  The *core* (also called *legacy*) group is found at REST path `/api/v1`.
+   The core group is not specified as part of the `apiVersion` field, for
+   example, `apiVersion: v1`.
+*  The named groups are at REST path `/apis/$GROUP_NAME/$VERSION` and use
+   `apiVersion: $GROUP_NAME/$VERSION` (for example, `apiVersion: batch/v1`).
+   You can find the full list of supported API groups in
+   [Kubernetes API reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/).
 
-The two paths that support extending the API with [custom resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/) are:
 
-- [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
-  for basic CRUD needs.
-- [aggregator](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md) for a full set of Kubernetes API semantics to implement their own API server.
+## API Extension
 
-## Enabling or disabling API groups
+The two paths that support extending the API with
+[custom resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/) are:
```

Comment:
I was tempted to fix the wording in this section. I still find the first sentence unwieldy.

Reply:
Yup. This sentence needs polishing. Here is how I'm gonna revise this.

- Promote this subsection to the API overview concepts page, considering that this subsection is small and a reader may need to know that there are ways to extend the API when they first read the API overview.
- Revise the content of the subsection so that the pointers are to the existing concepts pages (`/docs/concepts/extend-kubernetes/api-extension/custom-resources/` and `/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/`) rather than tasks pages.